toolbox/deps_update/main.go

```diff
@@ -119,12 +122,19 @@ func updateDeps(repo string, deps *[]u.Dependency, depChangeList *[]u.Dependency
 // which is auto-merged after presumbit
 func updateDependenciesOf(repo string) error {
 	log.Printf("Updating dependencies of %s\n", repo)
-	repoDir, err := u.CloneRepoCheckoutBranch(githubClnt, repo, *baseBranch, "")
+	saveDir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	repoDir, err := u.CloneRepoCheckoutBranch(githubClnt, repo, *baseBranch, "", "src/istio.io")
 	if err != nil {
 		return err
 	}
 	defer func() {
-		if err = u.RemoveLocalRepo(repoDir); err != nil {
+		if err = os.Chdir(saveDir); err != nil {
+			log.Fatalf("Error during chdir: %v\n", err)
+		}
+		if err = u.RemoveLocalRepo("src"); err != nil {
```

Comment:
I think `RemoveLocalRepo` takes an absolute path

Reply:
It just takes a path not absolute.